src/sensor/ping360.cpp

```diff
@@ -25,7 +25,7 @@ Q_LOGGING_CATEGORY(PING_PROTOCOL_PING360, "ping.protocol.ping360")
 
 Ping360::Ping360()
     :PingSensor()
-    ,_data(_num_points, 0)
+    ,_data(_sensorTimeout, 0)
```

Comment:
Did you mean to use `_sensorTimeout` to initialize here? That does not look correct.

Reply:
yep, should be `_maxNumberOfPoints`.